plugins/talk-plugin-featured-comments/client/components/ModActionButton.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import cn from 'classnames';
+import styles from './ModActionButton.css';
+import {pluginName} from '../../package.json';
+import {t} from 'plugin-api/beta/client/services';
+import {withTags} from 'plugin-api/beta/client/hocs';
+import {Icon} from 'plugin-api/beta/client/components/ui';
+
+export class Button extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      on: false
```

Comment:
The state should move to redux, just like in `talk-plugin-author-menu` and the reply box.
The current hiding/toggling behavior looks like (inconsistently) "working" because of a bug when rendering Slot elements. The logic when assigning keys to slot elements are incorrect and need to fix, when using _exculde if_. This causes this component to destruct and construct, thus starting with a fresh state.

Reply:
We can't move this to redux. We can't generate a store per button, (we have a button like this per comment) 🤔

Reply:
Look at how it's done in `talk-plugin-author-menu`, or the reply box.

Reply:
Sorry, wrong line, added the comment to the correct line 😅